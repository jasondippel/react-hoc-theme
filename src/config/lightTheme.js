export const VERSION = 1
export const TYPE = 'light'

export const VALUES = {
  /**********
   ** TEXT **
   **********/
  text: '#181A1B',
  textInverse: '#F5FAFC',
  textMeta: '#4e5356',
  textDisabled: '#61696b',
  link: '#007099',

  /****************
   ** BACKGROUND **
   ****************/
  background100: '#FFFFFF',
  background200: '#FAFAFA',
  background300: '#F1F1F1',
  background400: '#E3E7E8',
  background500: '#D9DDDE',
  backgroundLight: '#FFFFFF',
  backgroundDark: '#181A1B',
  keyline: '#CDD2D4',

  /***********************
   ** ACCENTS - PRIMARY **
   ***********************/
  primary: '#00AEEF',
  primaryAccent: '#B3E7FB',
  primaryBackground: '#DEF4FD',

  /***********************
   ** ACCENTS - SUCCESS **
   ***********************/
  success: '#7AAF2C',
  successAccent: '#C0DF95',
  successBackground: '#EEF7E2',

  /***********************
   ** ACCENTS - WARNING **
   ***********************/
  warning: '#f4b71e',
  warningAccent: '#FFE380',
  warningBackground: '#FFF1C0',

  /*********************
   ** ACCENTS - ERROR **
   *********************/
  error: '#FF1A1A',
  errorAccent: '#F9BFBF',
  errorBackground: '#FBE5E5',
}
