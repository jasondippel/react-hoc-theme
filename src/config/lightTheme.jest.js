import { VERSION, TYPE, VALUES } from './lightTheme'

describe('lightTheme', () => {
  test('version is unchanged', () => expect(VERSION).toEqual(1))

  test('type is light', () => expect(TYPE).toEqual('light'))

  test('values remain unchanged', () => expect(VALUES).toMatchSnapshot(VALUES))
})
