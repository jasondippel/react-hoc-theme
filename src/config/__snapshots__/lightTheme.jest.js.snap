// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lightTheme values remain unchanged 1`] = `
Object {
  "colors": Object {
    "background100": "#FFFFFF",
    "background200": "#FAFAFA",
    "background300": "#F1F1F1",
    "background400": "#E3E7E8",
    "background500": "#D9DDDE",
    "backgroundDark": "#181A1B",
    "backgroundLight": "#FFFFFF",
    "error": "#FF1A1A",
    "errorAccent": "#F9BFBF",
    "errorBackground": "#FBE5E5",
    "keyline": "#CDD2D4",
    "link": "#007099",
    "primary": "#00AEEF",
    "primaryAccent": "#B3E7FB",
    "primaryBackground": "#DEF4FD",
    "success": "#7AAF2C",
    "successAccent": "#C0DF95",
    "successBackground": "#EEF7E2",
    "text": "#181A1B",
    "textDisabled": "#61696b",
    "textInverse": "#F5FAFC",
    "textMeta": "#4e5356",
    "warning": "#f4b71e",
    "warningAccent": "#FFE380",
    "warningBackground": "#FFF1C0",
  },
}
`;
